examples-dev/voronoi.py

```diff
@@ -63,12 +64,12 @@ def generate_disk(partitions: int, depth: int):
     return np.column_stack((x, y)), triang.triangles
 
 
-def edge_plot(vertices, edge_nodes, ax, fill_value=-1, **kwargs):
+def edge_plot(vertices, edge_nodes, ax, **kwargs):
     n_edge = len(edge_nodes)
     edge_coords = np.empty((n_edge, 2, 2), dtype=float)
     node_0 = edge_nodes[:, 0]
     node_1 = edge_nodes[:, 1]
-    valid = (node_0 != fill_value) & (node_1 != fill_value)
+    valid = (node_0 != -1) & (node_1 != -1)
```

Comment:
Why not:

```suggestion
    valid = (node_0 != FILL_VALUE) & (node_1 != FILL_VALUE)
```

That clarifies that the edge is not connected to a node and a fill value is used. It also helps identifying parts of the code that could be affected when alterations are made to how fill values are treated.

Reply:
I had intended this more or less as a standalone-ish example. But you're right: FILL_VALUE is more clearly understood than -1.